src/Taboo/TabooRoom.py

```diff
@@ -95,9 +95,10 @@ def processMsg(self, qmsg):
             return self.__processJoin(qmsg)
 
         # When the last READY message is received, we should call
```

Comment:
I think we can get rid of the comment now that we have the functionality in place.
Or at least make it concise

Reply:
the next pull request has a more concise version that I will remove in the future. Marking this as resolved for now.